timbuctoo-instancev4/src/test/java/nl/knaw/huygens/timbuctoo/v5/jsonldimport/GenerateRdfPatchFromJsonLdEntityTest.java

```diff
@@ -0,0 +1,219 @@
+package nl.knaw.huygens.timbuctoo.v5.jsonldimport;
+
+import javaslang.collection.Map;
+import nl.knaw.huygens.timbuctoo.v5.bdbdatafetchers.dto.CursorQuad;
+import nl.knaw.huygens.timbuctoo.v5.dataset.Direction;
+import nl.knaw.huygens.timbuctoo.v5.filestorage.exceptions.LogStorageFailedException;
+import nl.knaw.huygens.timbuctoo.v5.rdfio.RdfPatchSerializer;
+import org.junit.Test;
+
+import javax.ws.rs.core.MediaType;
+import java.net.URI;
+import java.nio.charset.Charset;
+import java.util.HashMap;
+import java.util.stream.Stream;
+
+import static nl.knaw.huygens.timbuctoo.v5.util.RdfConstants.STRING;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+
+public class GenerateRdfPatchFromJsonLdEntityTest {
+  @Test
+  public void testGenerateAdditions() throws Exception {
+    Entity testEntity = ImmutableEntity.builder()
```

Comment:
You shouldn't expose ImmutableEntity. Instead it's better to add a static create() method in the interface similar to how it's done in `timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/security/dto/Login.java`

Reply:
You can do this fastest by selecting the values you insert and extracting them as variables (ctrl-alt-v) then selecting the builder statement and extracting the method (ctrl-alt-v) intellij should then ask you to extract in everywhere in the file. Then you can make the method static and move it to Entity (making it static is a refactoring, moving is done by putting your cursor on the name and pressing F6)